frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/initial-state/storage-tab-initial-state.ts

```diff
@@ -71,15 +70,15 @@ const getUrlStorage = (storageClassConfigMap: ConfigMapKind) => ({
   volume: VolumeWrapper.initializeFromSimpleData({
     name: ROOT_DISK_NAME,
     type: VolumeType.DATA_VOLUME,
-    typeData: { name: prefixedID(VM_TEMPLATE_NAME_PARAMETER, ROOT_DISK_NAME) },
+    typeData: { name: ROOT_DISK_NAME },
```

Comment:
can we replace ROOT_DISK_NAME with the generated name?

Reply:
Done! That was a really dumb mistake. I must have been more tired than I thought. =)